### PR TITLE
Fix assert messages

### PR DESCRIPTION
**Pull request by jschneidereit**

### CODE REVIEW

File: MsTestToXunitConverter.xUnit/AssertInvocationTests.cs [hunk 1]
```diff
@@ -19,6 +20,27 @@ public class AssertInvocationTests
             var target = pod.ActualRoot.GetInvocation();
 
             var new_root = pod.ActualRoot.ReplaceNode(target, sut(target));
+            
+            var actual = pod.ActualDocument.WithSyntaxRoot(new_root);
+            actual = await Formatter.FormatAsync(actual, annotation: annotation);
+            var expected = pod.ExpectedDocument;
+
+            var actual_string = actual.GetSyntaxRootAsync().Result.GetInvocation().ToString();
```
Comment (RikkiGibson) [inline MsTestToXunitConverter.xUnit/AssertInvocationTests.cs]: `ToEnvNewlineString`

File: MsTestToXunitConverter.xUnit/ResourceHelper.cs [hunk 2]
```diff
@@ -33,15 +33,17 @@ internal static class ResourceHelper
 
         internal static TestPod GetTestPod(string name)
         {
-            var actual_src = GetTestFile(name);
-            var actual = _baseProject.AddDocument(name: $"{name}.cs", text: actual_src, filePath: $"{name}.cs");
-            var actual_root = actual.GetSyntaxRootAsync().Result;
+            var actualSrc = GetTestFile(name);
+            var actualDoc = _baseProject.AddDocument(name: $"{name}.cs", text: actualSrc, filePath: $"{name}.cs");
+            var actualRoot = actualDoc.GetSyntaxRootAsync().Result;
```
Comment (RikkiGibson) [inline MsTestToXunitConverter.xUnit/ResourceHelper.cs:41]: `await`

File: MsTestToXunitConverter.xUnit/Resources/AssertInvocations/TestAreEqualMessage.cs [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+namespace MsTestToXunitConverter.xUnit
+{
+    class TestClass
+    {
+        public void TestAreEqualMessage()
+        {
+            Assert.AreEqual(1, 2, "poopy butthole");
```
Comment (RikkiGibson) [inline MsTestToXunitConverter.xUnit/Resources/AssertInvocations/TestAreEqualMessage.cs]: Jim I think you were working too late

**Comment by jschneidereit** [comment]
Going to merge this fundamentally broken code to prepare for v2

**Pull request merged**